distribution/server-x/src/main/content/bin/kc.sh

```diff
@@ -77,10 +86,11 @@ if [ "$DEBUG_MODE" = "true" ]; then
 fi
 
 CLASSPATH_OPTS="$DIRNAME/../lib/quarkus-run.jar:$DIRNAME/../lib/main/*"
+CONFIG_ARGS="-Dkeycloak.config.args=${CONFIG_ARGS#?}"
```

Comment:
This is an implementation detail so that we can handle arguments without necessarily implementing code in scripts. It will be processed and transformed internally to the corresponding properties.

Reply:
Yes, I see. I was wondering that this is not exposed to the end users. Thanks!